examples/platform/linux/BUILD.gn

```diff
@@ -96,6 +96,7 @@ source_set("app-main") {
     ":smco-test-event-trigger",
     ":water-heater-management-test-event-trigger",
     "${chip_root}/src/app/codegen-data-model-provider:instance-header",
+    "${chip_root}/src/controller:controller",
```

Comment:
I'm looking into the nrfconnect failure (seems like the part that failed was skipped in the PR builds for #36567).
Removing this dependency was one of the goals of that change though.